frontend/app/components/auth-panel/__anonymous-login-form/auth-panel__anonymous-login-form.tsx

```diff
@@ -69,6 +87,7 @@ export class AnonymousLoginForm extends Component<Props, State> {
 
   render() {
```

Comment:
I like preact way to get props :) If you too it could be like this.
```suggestion
  render({ intl, ...props }) {
```

Reply:
Let's do it in separate pull request.